test/test_va_api_fixture.h

```diff
@@ -126,6 +158,8 @@ class VAAPIFixture : public ::testing::Test {
     std::vector<VAConfigAttrib> m_queryConfigAttribList;
     std::vector<VASurfaceAttrib> m_querySurfaceAttribList;
     std::vector<VASurfaceID> m_surfaceID;
+    std::vector<VAImageFormat>m_imageFmtList;
+    std::vector<VAImageFormat>m_subPicFmtList;
```

Comment:
please add space between type and variable name

Reply:
Updated.